Mathlib/RingTheory/PowerSeries/Order.lean

```diff
@@ -137,7 +137,7 @@ theorem min_order_le_order_add (φ ψ : R⟦X⟧) : min (order φ) (order ψ) 
   refine le_order _ _ ?_
   simp +contextual [coeff_of_lt_order]
 
-@[deprecated (since := "2024-11-12")] alias le_order_add := min_order_le_order_add
+@[deprecated (since := "2024-11-30")] alias le_order_add := min_order_le_order_add
```

Comment:
This doesn't seem to do what the PR title promises. Did you intend for a larger diff?

Reply:
I changed the date when I mistakenly thought I had modified that alias myself.